### PR TITLE
cpu/native: allow for multiple netdev2_tap devices

### PR DESCRIPTION
**Pull request by miri64**
Adaptation of #5614 with start-up sequence. Tested with `gnrc_networking` with

```bash
PORT="tapX tapY"
CFLAGS="-DNETDEV2_TAP_MAX=2 -DGNRC_NETIF_NUMOF=2"
```

Currently the initialization of the (absolute) number of devices is fixed to `NETDEV2_TAP_MAX`, but with a clean-up of the argument parsing in native we could fix that. I leave this to a follow-up PR.

### CODE REVIEW

File: cpu/native/include/netdev2_tap_params.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
```
Comment (LudwigKnuepfer) [inline cpu/native/include/netdev2_tap_params.h]: Year has been updated upstream.

File: cpu/native/startup.c [hunk 2]
```diff
@@ -246,15 +247,15 @@ __attribute__((constructor)) static void startup(int argc, char **argv)
 
 #if defined(MODULE_NETDEV2_TAP)
     if (
-            (argc < 2)
+           (argc < (NETDEV2_TAP_MAX + 1)) /* one arg per tap + 0 for command */
```
Comment (LudwigKnuepfer) [inline cpu/native/startup.c]: This seems to apply the online help above needs updating as it currently says:
```c
#if defined(MODULE_NETDEV2_TAP)
    real_printf(" <tap interface>");
#endif
```

**Comment by kYc0o** [comment]
Tested on OS X and works correctly.

**Comment by miri64** [comment]
Addressed @LudwigKnuepfer's comments.

File: cpu/native/startup.c (hunk 2 above)
Comment (miri64) [inline cpu/native/startup.c]: (addressed)

**Comment by miri64** [comment]
Rebased to current master (and squashed because unpicking my stuff and then repicking was annoying)

**Comment by miri64** [comment]
Fixed all issues reported by the CIs and squashed immediately

**Comment by miri64** [comment]
Both CIs are happy :-)

**Comment by kYc0o** [comment]
Let's go then!

**Pull request merged**